### PR TITLE
fix: backend node modules version

### PR DESCRIPTION
**Pull request by jeonjonghyeok**
After testing, some modifications were made.

- Version correction for knex connection failure error has been reflected. #25 

before
```
package.json
"knex": "^0.20.10",
"pg": "^7.18.2"
```
after
```
package.json
"knex": "^0.21.1",
"pg": "^8.0.3"
```
You can refer to the link.
https://stackoverflow.com/questions/40435315/knex-timeout-acquiring-a-connection-the-pool-is-probably-full-are-you-missing

- I commented because github.html cannot be executed on the frontend.

- Updated README for easier execution.

### CODE REVIEW

File: packages/frontend/src/index.jsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ import { ToastProvider } from 'react-toast-notifications'
 import Group from './pages/group'
 import About from './pages/about'
 
-import GithubCorner from './components/github.html'
+//import GithubCorner from './components/github.html'
```
Comment (ChihChengLiang) [inline packages/frontend/src/index.jsx]: Let's remove it completely
```suggestion
```

**Review by ChihChengLiang — COMMENTED** [review]
Hey @jeonjonghyeok,

Thanks for the contribution. I left some comments. 

Another question, why is `scripts/buildImagesGoerli.sh` an empty file?

File: packages/frontend/src/index.jsx (hunk 1 above)
Comment (jeonjonghyeok) [reply packages/frontend/src/index.jsx]: Added execute permission to run the file!

**Comment by jeonjonghyeok** [comment]
> Hey @jeonjonghyeok,
> 
> Thanks for the contribution. I left some comments.
> 
> Another question, why is `scripts/buildImagesGoerli.sh` an empty file?

Added execute permission to run the file!

**Comment by ChihChengLiang** [comment]
Thank you @jeonjonghyeok for the contribution!

**Comment by jeonjonghyeok** [comment]
I'm honored to accept my first contribution
Thank you 😊

**Pull request merged**